tests/Integration/Subscriber/Addons/CloudflareSubscriber/TestSetVarnishLocalhost.php

```diff
@@ -76,8 +92,8 @@ public function testShouldReturnLocalhostWhenFilterTrue() {
 		$cf_subscriber     = new CloudflareSubscriber( new Cloudflare( new Options_Data( (new Options( 'wp_rocket_'))->get( 'settings' ) ), $cloudflare_facade ), new Options_Data( (new Options( 'wp_rocket_'))->get( 'settings' ) ), new Options() );
 
 		$this->assertSame(
-			'localhost',
-			$cf_subscriber->set_varnish_localhost( '' )
+			[ 'localhost' ],
+			$cf_subscriber->set_varnish_localhost( [] )
```

Comment:
@Tabrisrp Instead of directly invoking the method, find a way to test in an integrated fashion.